scripts/publish_prod.sh

```diff
@@ -86,6 +86,7 @@ echo "Publishing to https://pypi.org/project/datadog-lambda/"
 
```

Comment:
<img width="653" alt="Screenshot 2023-06-22 at 1 26 50 PM" src="https://github.com/DataDog/datadog-lambda-python/assets/49878080/38e92fb9-c14f-4809-aae6-76d80f090cb0">
see above we use the $NEW_VERSION var